pkg/resiliency/resiliency.go

```diff
@@ -618,7 +618,7 @@ func (r *Resiliency) ActorPreLockPolicy(ctx context.Context, actorType string, i
 	var t time.Duration
 	var rc *retry.Config
 	var cb *breaker.CircuitBreaker
-	operationName := fmt.Sprintf("actor[%s, %s]", actorType, id)
+	operationName := "actor[" + actorType + ", " + id + "]"
```

Comment:
Same, not a huge deal but we don't do concatentation elsewhere really that I have noticed

Reply:
Same, this is a very hot path